tests/ui/landing_page_test.py

```diff
@@ -24,6 +24,12 @@ def setUp(self):
   def tearDown(self):
     """Teardown for test methods."""
     landing_page = LandingPage(self.driver)
+    try:
+      user_details_button = landing_page.get_element(
```

Comment:
The issue was more so with the actual click portion I believe, not getting the element so I don't see the need.